README.md

```diff
@@ -24,6 +24,14 @@ This plugin creates 3 buttons for the first 3 scripts in your `package.json` and
 
 You can control if a command should end up as a button or in the list by rearranging the scripts in your `package.json`
 
+## Long running tasks
```

Comment:
Can you please change the title to `Configuration`
just like in https://github.com/jadjoubran/webdash-performance-budget#configuration

End users will have this configuration in `webdash.json` already (we'll work on a fix for it on the main repo later)

I just realized that `longRunningScripts` means that they just take too long to finish
so how about we call them `serverScripts`?

Reply:
agree with changing longRunningScripts to serverScripts 👍
for Configuration there already exists a title, i can remove Long running tasks or write configuration for sever scripts instead, what's your opinion ?

Reply:
yes please add it at the end of #Configuration